qiskit/transpiler/__init__.py

```diff
@@ -28,8 +29,8 @@
 resulting outcomes.  Rewriting quantum circuits to match hardware constraints and
 optimizing for performance can be far from trivial.  The flow of logic in the rewriting
 tool chain need not be linear, and can often have iterative sub-loops, conditional
-branches, and other complex behaviors.  That being said, the basic building blocks
-follow the structure given below:
+branches, and other complex behaviors. That being said, the standard
+compilation operates in fixedfollow the structure given below:
```

Comment:
"fixedfollow"?

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9087/commits/a46e97de3261a7a394c0f63ff23f818f5936f49f